all_in_one.ts

```diff
@@ -1,21 +1,19 @@
 import dotenv from 'dotenv'
```

Comment:
In the `all_in_one.ts` file you execute all of the `pre-steps/` operations from one unified file.
For future reference: is there a cleaner and more concise way how you could have imported the .ts files inside of the `pre-steps` directory as modules and have invoked their functions instead of copy and pasting their source code into a unified file?